src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -640,6 +640,7 @@ export default class ListPeopleAndRoles extends Mixins(CommonMixin, OrgPersonMix
 .actions-width {
   min-width: 150px;
   max-width: 150px;
+  padding: 0.35rem !important;
```

Comment:
I'm thinking you don't don't all-around padding, just the top. And maybe top margin will work better?

Also, "0.35rem" is a value we haven't used before. Typically we use "Vuetify units", which are quarter-rem (4px). Or sometimes we bump buttons or icons up or down 1 or 2 pixels.

Also, mouse over the button and drop-down arrow and make sure the padding is the same as the other right-side buttons on this page.

Also, does this affect other filings, and what do they look like now?